src/framework/theme/components/layout/layout.component.ts

```diff
@@ -205,8 +205,9 @@ export class NbLayoutComponent implements AfterViewInit, OnDestroy {
     this.restoreScrollTopValue = convertToBoolProperty(val);
   }
 
-  @ViewChild('layoutTopDynamicArea', { read: ViewContainerRef }) veryTopRef: ViewContainerRef;
-  @ViewChild('scrollableContainer', { read: ElementRef }) scrollableContainerRef: ElementRef;
+  // TODO remove as of 5.0.0
```

Comment:
Probably we should also add issue for tests refactoring?

Reply:
There is a note here https://github.com/akveo/nebular/issues/1514

Reply:
Ok then please refer the issue as you did in other comments 🙂